Dockerfile

```diff
@@ -69,8 +69,6 @@ RUN set -e \
     && apt install -y \
         libreadline-dev \
         libseccomp-dev \
-        libicu67 \
-        openssl \
```

Comment:
NB: the `Dockerfile.compute-node` still uses the Debian packages and links dynamically

https://github.com/neondatabase/neon/blob/f67010109f488cd20e7aae9b8feaf0c2016b63e9/Dockerfile.compute-node#L12-L16

Reply:
Yes, previously our focused only for Pageserver. if we want to do the same for compute then we can remove it from here and use it from build-tools image.

Reply:
Punting this to https://github.com/neondatabase/cloud/issues/14519